tests/test_jetpack.py

```diff
@@ -5,53 +5,60 @@
 import nose
 
 from js_helper import _do_real_test_raw as _js_test
-from validator.testcases.markup.markuptester import MarkupParser
-import validator.testcases.jetpack as jetpack
+from tests.test_frameworks import (MockXPI, accepted_version,
+                                   assert_framework_files, disposable_metadata)
+from validator import constants
 from validator.errorbundler import ErrorBundle
+from validator.testcases import content, libraries
+from validator.testcases.markup.markuptester import MarkupParser
 from validator.xpi import XPIManager
 
 
-def _do_test(xpi_package, allow_old_sdk=True):
+# These files are a subset of the framework tests which are specific
+# to Jetpack. They're currently split into a separate file since
+# they're considerably more comprehensive than any other framework
+# tests, and mostly predate the current generalized tests.
 
-    err = ErrorBundle()
-    jetpack.inspect_jetpack(err, xpi_package, allow_old_sdk=allow_old_sdk)
-    return err
+with open("tests/resources/bootstrap.js") as bootstrap_file:
+    bootstrap = bootstrap_file.read()
+with open("tests/resources/bootstrap-jpm.js") as bootstrap_file:
+    bootstrap_jpm = bootstrap_file.read()
 
 
-class MockXPI(object):
+def _do_test(xpi_package, allow_old_sdk=True):
 
-    def __init__(self, resources):
-        self.resources = resources
+    err = ErrorBundle()
+    content.test_packed_packages(err, xpi_package)
 
-    def read(self, name):
-        if isinstance(self.resources[name], bool):
-            return ""
-        return self.resources[name]
+    if allow_old_sdk:
+        ignore_ids = [
```

Comment:
If I changed it to a tuple rather than a list, and changed the `in` to `==`. I implemented it this way since it didn't seem to add significant complexity, and it seemed likely we'd want to add other IDs in the future.